src/domain/login/LoginViewModel.ts

```diff
@@ -193,20 +206,17 @@ export class LoginViewModel extends ViewModel<Options> {
         );
     }
 
-    private _disposeViewModels(): void {
-        this._startSSOLoginViewModel = this.disposeTracked(this._startSSOLoginViewModel);
-        this._passwordLoginViewModel = this.disposeTracked(this._passwordLoginViewModel);
-        this._completeSSOLoginViewModel = this.disposeTracked(this._completeSSOLoginViewModel);
-        this.emitChange("disposeViewModels");
+    updateLoginOptions(): void {
```

Comment:
```suggestion
    private _updateLoginOptions(): void {
```
Don't forget to add the "_" to where this method is called.